styles/ink.less

```diff
@@ -244,3 +244,42 @@ atom-text-editor::shadow {
     font-weight: bold;
   }
 }
+
+.ink-tooltip {
```

Comment:
Is ink-tooltip specific to progress bars? If so it probably could have a name that reflects that; if not it'd be better to split it out and make it truly general.